fableous-fe/src/Auth.tsx

```diff
@@ -0,0 +1,20 @@
+import { AxiosResponse } from "axios";
+
+const TOKEN_KEY = "token";
+
+const auth = {
+  saveToken: (token: AxiosResponse<any>) => {
+    localStorage.setItem(TOKEN_KEY, token.headers.authorization);
```

Comment:
one more thing @wahyuadt , in this file u save the token "Bearer abcdef..." as is, that exact format (with Bearer) is needed for rest API requests, but for websocket requests, the only thing needed it the token part (WITHOUT Bearer), so I think its better to save only the "abcdef..." part, and in interceptor, append it to "Bearere "